src/webapi/rendering_context.rs

```diff
@@ -60,36 +60,62 @@ pub struct ImageData(Reference);
 #[reference(instance_of = "TextMetrics")]
 pub struct TextMetrics(Reference);
 
-/// The type of compositing operation to apply when drawing new shapes, where type is a string 
-/// identifying which of the twelve compositing operations to use.
-/// 
+/// The type of compositing operation to apply when drawing new shapes
+///
 /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation)
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub enum CompositeOperation {
+    /// Draws new shapes over existing canvas content
     SourceOver,
+    /// Draws new shapes over existing canvas content, but only where existing canvas content overlaps. Anything not in that area becomes transparent.
     SourceIn,
+    /// New shapes are drawn where there is no existing canvas content. Everything else (including old canvas content) is made transparent.
     SourceOut,
+    /// Draws new shapes only where there is existing canvas content, over existing canvas content
     SourceAtop,
+    /// Draws new shapes behind existing canvas content
     DestinationOver,
+    /// Keeps existing canvas content where it overlaps with the new shape. Everything outside the overlap is made transparent.
     DestinationIn,
+    /// The existing content is kept where it doesn't overlap with the new shape. Everything else, including the new shape area, is made transparent.
+    DestinationOut,
```

Comment:
Since introducing an enum value is technically a breaking change can you comment this line out and leave a TODO? I'll uncomment it once I'll be making the next semver-incompatible change.